tailwind.config.js

```diff
@@ -244,6 +245,10 @@ module.exports = {
         purple: "#353552",
       },
     },
+    boxShadow: {
```

Comment:
@jhonatan-lopes Do you know if these "pop" and "outline" styles are also used elsewhere on the site? Or if they are defined in the design system? I'm a little worried that we add these styles as "boxShadow/pop/outline" to Tailwind configs because they are very specifically to this use case, and there are many other existing elements on the site have some sort of box shadow that we eventually need to make them adapt to Tailwind classes. If we want to keep these in the config, maybe rename these classnames to something less general could be an option?

Reply:
I see your point @mmmavis , thanks! I got the names from the design team, so I think these would be reused on the blog as it is defined with their design system. In the end, I thought this was "cleaner" than one time styles, but maybe I could rename them to be on a blog scope, e.g. "blog-pop" and "blog-outline"? What do you think?

Reply:
Here it is on the [design system](https://www.figma.com/file/528nyaJFzvCXAiw2gglwzh/Blog-Improvements?node-id=627%3A22439&mode=dev):
![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/61277874/b99bc3d2-62c2-4ac7-af18-021101600b27)